src/main/java/codeu/controller/RegisterServlet.java

```diff
@@ -0,0 +1,19 @@
+package codeu.controller;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+* Servlet class responsible for user registration.
```

Comment:
Nit: this comment just restates the identifier name in text. I see this everywhere! Can you think of something to put here that is concise, but will speed up a new developer's understanding of this class?

I'm a bit unconventional in that I believe you can leave out comments in cases like this.